origin-growth/src/scripts/verifyEvents.js

```diff
@@ -12,6 +12,14 @@ const parseArgv = require('../util/args')
 Logger.setLogLevel(process.env.LOG_LEVEL || 'INFO')
 const logger = Logger.create('verifyEvents', { showTimestamp: false })
 
+// Load the production fraud engine if available, and fall back to the dev one otherwise.
+let FraudEngine
+try {
+  FraudEngine = require('../fraud/prod')
```

Comment:
I would prefer a hard fail in incorrect environment. This way it can not fallback to dev on production: 
```
if (process.env.GROWTH_FRAUD_DETECTION) {
  FraudEngine = require('../fraud/prod')
} else {
  FraudEngine = require('../fraud/dev')
}

```

Reply:
I think that @sparrowDom is probably right, it is better to hard fail if something goes wrong with pulling the production code. We could use `process.env.NODE_ENV === 'production'` so we don't have to introduce another environment variable. Maybe also add a few logs at this point to say something like `Loaded development fraud engine`, or `Loaded production fraud engine`.

Reply:
I was thinking of the `process.env.NODE_ENV === 'production'` solution also, but was concerned that setting that as an environmental variable might affects some other configuration in the container which might have undesired side effects.

Reply:
Good point. Done.

Reply:
NODE_ENV is already set at the container level (see [this](https://github.com/OriginProtocol/origin/blob/master/deployment/dockerfiles/origin-growth#L5)) so let me use that...

Reply:
Sorry I wasn't clear of what I meant to say. For production purposes `NODE_ENV` is perfect, for local setup (few developers who will have that) only `NODE_ENV` might cause side effects. 

Perhaps something like: 
```
if (process.env.NODE_ENV === 'production' || process.env.GROWTH_FRAUD_DETECTION)  {
...
```
So on local we can just set the `GROWTH_FRAUD_DETECTION` to true without the risk of side effects of setting `NODE_ENV = true`

Reply:
Sounds good. Added.